### ISSUE TITLE
[Feature Request] Much higher or configurable maxScale for ZoomableImageView?

### ISSUE THREAD

**Issue by kylepaulsen** [hardcore-sushi/DroidFS#275]
Hey this is a super cool app! I had a feature request:

It would be nice for large images with text to be zoomable up to a very large scale - maybe like 10x? Right now it looks like the max scale is 3: https://github.com/hardcore-sushi/DroidFS/blob/967d4551c57abbc9febd5f23f6a7fd1af4118b89/app/src/main/java/sushi/hardcore/droidfs/widgets/ZoomableImageView.java#L27C24-L27C32

Either a larger value like 10 or some way for the user to configure that would be awesome.

**Comment by hardcore-sushi** [comment]
Yes you are right. This limit should only be used for double-tap zoom, not for manual pinch zoom.

**Comment by hardcore-sushi** [comment]
#392 removes this limit for manual pinch zoom 🚀
